cockatrice/src/messagelogwidget.cpp

```diff
@@ -341,7 +341,7 @@ void MessageLogWidget::logDrawCards(Player *player, int number)
         mulliganPlayer = player;
     else {
         soundEngine->playSound("draw_card");
-        appendHtmlServerMessage(tr("%1 draws %2 card(s).")
+        appendHtmlServerMessage(tr("%1 draws %2 card(s).", "", number)
```

Comment:
IIRC in order to work this string needs to contain a `%n` in place of `%2`.

Reply:
nope, it works. i've compiled cockatrice and tested those changes.

Reply:
The documentation has %n, too:
http://doc.qt.io/qt-5/i18n-source-translation.html#handling-plurals

Maybe transifex needs it to properly display the various translations options?

Reply:
I think transifex just reads the numerous and numerusform tags from the ts file to handle that but i don't know for sure

Reply:
The `%n` can be used, but doesn't have to, all instances of `%n` would be converted to the value of `number`. We can simply pass the number a second time as an `.arg()`, that way we can easily pass the font color html tags as well.